modules/local/bedtools_genomecov.nf

```diff
@@ -2,7 +2,7 @@
 include { initOptions; saveFiles; getSoftwareName } from './functions'
 
 params.options = [:]
-def options    = initOptions(params.options)
+options        = initOptions(params.options)

```

Comment:
These are mostly updates related  to upstream changes in modules correct?

Reply:
Yup. That's correct. Need to remove them because it messes with the `-resume` functionality.